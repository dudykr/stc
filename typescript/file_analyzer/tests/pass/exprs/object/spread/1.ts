declare let o: object;
export const x: { a: number, b: number } = { a: 1, ...o, b: 2 };
