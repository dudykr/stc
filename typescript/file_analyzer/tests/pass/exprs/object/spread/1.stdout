warning: Type
 --> $DIR/tests/pass/exprs/object/spread/1.ts:2:55
  |
2 | export const x: { a: number, b: number } = { a: 1, ...o, b: 2 };
  |                                                       ^
  |
  = note: object

warning: Type
 --> $DIR/tests/pass/exprs/object/spread/1.ts:2:44
  |
2 | export const x: { a: number, b: number } = { a: 1, ...o, b: 2 };
  |                                            ^^^^^^^^^^^^^^^^^^^^
  |
  = note: object

